HoloStorageConnector/HoloRpository/Demo/Scripts/DemoScript.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Collections;
```

Comment:
![image](https://user-images.githubusercontent.com/11090412/62306058-0a65d880-b479-11e9-8139-7cd5f4761a6e.png)
I can't really get the DemoScene to do anything. I have the server running on port 3001 but the buttons are not there so I can't really do anything or load any data?

Please refer to what we talked about, this needs to be fixed, and also all models/prefabs etc. you find somewhere in the MRTK examples sub-directories can be copied to a main directory of your project

Reply:
I have moved the prefab to the package, and modified the scene, plz have a try again

Reply:
![image](https://user-images.githubusercontent.com/11090412/62319277-866e1980-b495-11e9-8a2f-423672f0ddbe.png)

Still nothing. Still shows Missing Mesh for the Tube, Cylinder...
![image](https://user-images.githubusercontent.com/11090412/62319257-7b1aee00-b495-11e9-9c99-41276d9ea632.png)

Maybe you have to google this issue, might be some Unity quirk.. Also you could ask Kawai if it works on his computer

Reply:
Update: I imported the MRTK v2 to a new project and followed [these steps](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/GettingStartedWithTheMRTK.html). When I open the demo scene there, I guess everything looks as it should? So it may not be my PCs problem then...
  
![image](https://user-images.githubusercontent.com/11090412/62320836-2b3e2600-b499-11e9-8222-8c844df02b66.png)

Reply:
Update: I manually imported MRTK (RC2) Examples package, and now it works. TODO: Find a solution... either include all necessary files in our repo (like I said, maybe copying them to the folders directly in the project), or alternatively nothing from the MRTK, and make it a requirement for devs to go and import it...?

![image](https://user-images.githubusercontent.com/11090412/62324206-e28a6b00-b4a0-11e9-8307-5a89c08178f2.png)

Reply:
Update: still no buttons. But even though they are invisible, I can interact with them.
![image](https://user-images.githubusercontent.com/11090412/62376321-4c098880-b538-11e9-9f44-c9a2af84edf8.png)

Reply:
Update: Solved. On my computer, needed to manually re-import MRTK Foundations + Examples